packages/core/getUserAvailability.ts

```diff
@@ -123,8 +133,14 @@ export const getCurrentSeats = async (
   return monitorCallbackAsync(_getCurrentSeats, ...args);
 };
 
-const _getCurrentSeats = async (eventTypeId: number, dateFrom: Dayjs, dateTo: Dayjs) => {
-  return await prisma.booking.findMany({
+const _getCurrentSeats = async (
+  eventTypeId: number,
+  dateFrom: Dayjs,
+  dateTo: Dayjs,
+  isTeamEvent?: boolean,
+  hosts?: string[]
```

Comment:
`_getCurrentSeats` now takes in 5 parameters, which is a lot for a function. I propose combining `eventTypeId`, `isTeamEvent` and `hosts` into 1 `eventType` parameter:
```
eventType: Pick<EventType, "id" | "schedulingType" | "hosts">,
dateFrom: Dayjs,
dateTo: Dayjs
```

and then within `_getCurrentSeats` we access event type properties to check:
```
const isTeamEvent = eventType?.schedulingType !== null;
const hostEmails = eventType?.hosts.map((host) => host.user.email);
```

And to invoke function:
```
currentSeats = await getCurrentSeats(eventType, dateFrom, dateTo);
```